docs/releasing/publishing.md

```diff
@@ -2,7 +2,7 @@
 
 Read the docs for [what to do before publishing a release](/docs/releasing/before-publishing-a-release.md) to ensure you are prepared to publish.
 
-See the [documentation on support branches](https://govuk-design-system-team-docs.netlify.app/how-we-work/version-control/support-branches.html#support-branches) if you need to:
+See the [documentation on support branches](https://govuk-design-system-team-docs.netlify.app/guides/version-control/support-branches/#support-branches) if you need to:
```

Comment:
Might as well update the domain whilst we're here?

```suggestion
See the [documentation on support branches](https://team-playbook.design-system.service.gov.uk/guides/version-control/support-branches/#support-branches) if you need to:
```

Or I can check for other instances in a follow up PR if you prefer